account_invoice_customer_no_autofollow/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,2 @@
+Go to General Settings/Invoicing;
```

Comment:
Go to General Settings/Invoicing (or Accounting)

Reply:
Got it, I'll fix it